tests/integ/kms_utils.py

```diff
@@ -12,19 +12,26 @@
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
 
+import json
+
 from botocore import exceptions
 
-KEY_ALIAS = "SageMakerIntegTestKmsKey"
+PRINCIPAL_TEMPLATE = '["{account_id}", "{role_arn}", "arn:aws:iam::{account_id}:role/{' \
+                     'sagemaker_role}"] '
+
+KEY_ALIAS = "SageMakerTestKMSKey"
+KMS_S3_ALIAS = "SageMakerTestS3KMSKey"
+POLICY_NAME = "default"
 KEY_POLICY = '''
 {{
   "Version": "2012-10-17",
-  "Id": "sagemaker-kms-integ-test-policy",
+  "Id": "{id}",
```

Comment:
single quotes in ll. 22-24 and 28

Reply:
Changed lines 22-24. Line 28 is JSON, single quotes are not valid.